specs/validator-system-v2/overview.md

```diff
@@ -0,0 +1,107 @@
+# Validator System V2
+
+<!-- All glossary references in this file. -->
+
+[g-l2-output]: ../glossary.md#l2-output-root
+[g-zk-fault-proof]: ../glossary.md#zk-fault-proof
+[g-validator-pool-contract]: ../glossary.md#validator-pool-contract
+[g-validator]: ../glossary.md#validator
+[g-validator-reward]: ../glossary.md#validator-reward
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Background](#background)
+- [Overview](#overview)
+  - [Participants](#participants)
+    - [Validators](#validators)
+    - [KRO Delegators](#kro-delegators)
+    - [KGH Delegators](#kgh-delegators)
+  - [Priority Validator Selection](#priority-validator-selection)
+  - [Reward Distribution](#reward-distribution)
+- [Contracts](#contracts)
+- [Summary of Definitions](#summary-of-definitions)
+  - [Constants](#constants)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Background
+
+The [current permissionless validator system](../protocol/validator.md) of Kroma ensures economic security by bonding
+`REQUIRED_BOND_AMOUNT` of ETH each time the validator submits an [L2 output root][g-l2-output]. While successfully
+involving over 360 validators (as of 6/3/24), there are a few drawbacks in the current system that need to be addressed:
+
+- **Validator Exploitation Vulnerability**: Since the probability of being selected as a priority validator is
```

Comment:
I think the `Exploitation Vulnerability` is kinda strong word and implies that the current system has a serious bug, so how about` Multiple Accounts on a Single Validator `?